requirements.txt

```diff
@@ -1,4 +1,5 @@
-pytest==6.1.2
```

Comment:
`tests/requirements.txt` file should be moved to `test-requirements.txt` under repository root. Don't forget to update ci script accordingly.